### PR TITLE
<PackageSidebar /> component tests

### PR DESCRIPTION
**Pull request by ayusharma**
**Type:** bug / feature / documentation / unit test / build

The following has been addressed in the PR:

*  Unit or Functional tests are included in the PR

**Description:**

 
Package sidebar component test cases.

### CODE REVIEW

File: test/webui/components/PakageSidebar/maintainers.spec.js [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+  it('should get all the contributors with false for showAllContributors', () => {
+    expect(instance.showAllContributors).toBeFalsy();
+  });
+
+  it('should get unique contributors', () => {
+    const result = [
+      {
+        avatar:
+          'https://www.gravatar.com/avatar/4ef03c2bf8d8689527903212d96fb45b',
+        email: 'chocolatey030@gmail.com',
```
Comment (juanpicado) [inline test/webui/components/PakageSidebar/maintainers.spec.js]: please, let use fake emails ... we do not want piss off someone with smap :)

Comment (ayusharma) [reply test/webui/components/PakageSidebar/maintainers.spec.js]: I'll change emails.

**Comment by ayusharma** [comment]
@juanpicado @Meeeeow Please review.

**Comment by juanpicado** [comment]
@Meeeeow I have to merge this, looks good for me, let me know whether something was wrong.

**Pull request merged**